src/stream.c

```diff
@@ -313,6 +313,8 @@ int start_play(streams *sid, sockets *s)
 		s->flush_enqued_data = 1;
 	sid->do_play = 1;
 	sid->start_streaming = 0;
+	if (s->type != TYPE_HTTP)
```

Comment:
Probably no point in differentiating between stream types here, so you can delete this.

Reply:
Sorry, in this version one delete is missing.

This check is required, because for HTTP requests this code is executed in every request. So after the first request the next one doesn't needs to reset the `start_streaming`. For HTTP the initialization is done in line 353:
https://github.com/catalinii/minisatip/pull/580/files#diff-b4c34cf7e2b840ebe24729f28a8c5f27R353

Reply:
The missing deletion (which has confused you) is now here:
https://github.com/catalinii/minisatip/pull/580/files#diff-b4c34cf7e2b840ebe24729f28a8c5f27L315